source/http_message.c

```diff
@@ -115,20 +115,6 @@ struct aws_http_message *aws_napi_http_message_unwrap(napi_env env, napi_value j
  * Constructor
  **********************************************************************************************************************/
 
-static void s_napi_http_request_finalize(napi_env env, void *finalize_data, void *finalize_hint) {
```

Comment:
There were 2 different finalizers for the same class. I combined them into 1.

(When this code was first written, the 2 different finalizers were necessary because the underlying C classes didn't have reference-counting. But now they do, so the weirdness the led to needing 2 different finalizers isn't necessary anymore)